scenarioo-client/app/pipes/dateTime.pipe.spec.ts

```diff
@@ -0,0 +1,52 @@
+/* scenarioo-client
+ * Copyright (C) 2014, scenarioo.org Development Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {DateTimePipe} from './dateTime.pipe';
+
+declare var angular: angular.IAngularStatic;
+
+describe('Pipe: scDateTime', () => {
+    let scDateTime: DateTimePipe;
+    let input;
+
+    // Arrange: set up new instance of the pipe DateTime
+    beforeEach(() => {
+        scDateTime = new DateTimePipe('en-US');
+    });
+
+    it('Undefined returns an empty string', async () => {
+        // Act
+        input = scDateTime.transform(undefined);
```

Comment:
input is actually output or I don't get it. You put input into your pipe and get output which you then assert

Reply:
You're right, thanks.